packages/yoga/src/Heading/web/BackButton.jsx

```diff
@@ -1,26 +1,78 @@
 import React from 'react';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 import PropTypes from 'prop-types';
 import { ArrowLeft } from '@gympass/yoga-icons';
 import Button from '../../Button';
+import Box from '../../Box';
+import Icon from '../../Icon';
 
-const ButtonIcon = styled(Button.Icon)`
+const ButtonIcon = styled(Button)`
```

Comment:
What do you think about the approach bellow where i'm using the `secondary` and `inverted` props from `Button.Icon`? This way we only need to override some `Button.Icon` styles.

https://github.com/gympass/yoga/pull/735/files#diff-d4cfcfeaeb3e6306f1915918e710fcae9615d3fa12d829e229ee1684962e33d8

Reply:
hmm... maybe in my case it won't work... because I have a visible area and a clickable area with different sizes, this is why I'm adding the `Icon` inside the `Button` and working on the sizes.

[You can see it here](https://www.figma.com/file/Hsrgmg4oLjJxYcJHaFeuJP/%F0%9F%8E%A8-Yoga-3?type=design&node-id=2203-354&mode=design&t=Ewh5gxYSMxZyFkmN-4)

Reply:
I hadn't noticed this click area, thanks!